packages/core/src/forms/control/control.element.ts

```diff
@@ -280,6 +280,8 @@ export class CdsControl extends LitElement {
   updated(props: Map<string, any>) {
     super.updated(props);
     this.messages.forEach(message => syncProps(message, this, { disabled: props.has('disabled') }));
+
+    syncProps(this.inputControl, this, { disabled: props.has('disabled') });
```

Comment:
We should add a test to check this, in the control group tests you can use a test group with a child control to set the group and ensure the child was also set similar to this https://github.com/vmware/clarity/blob/next/packages/core/src/forms/control-group/control-group.element.spec.ts#L81

Reply:
Okay.

I am also trying to add a test in `control-group.element` but I can't find a way the `input` to receive a `disabled` attribute. 

`controlGroup.disabled = true` didn't work
`controlGroup.update(new Map().set('disabled' , true))` didn't work as well

Reply:
![image](https://user-images.githubusercontent.com/15037947/111333480-69428a80-867b-11eb-834e-9f15eef56fcd.png)

Reply:
Its likely because lit renders async, while you have an await on the controlGroup it would also need to await on the child cds-control inputs that would asynchronously be setting the disabled attr on the input

Reply:
This wont test the group use case but this will test that the control does set the native input correctly which I think is good for this fix. 

```javascript
it('should disable native input if cds-control is set disabled (set by cds-control-group)', async () => {
    await componentIsStable(control);
    expect(input.hasAttribute('disabled')).toBe(false);

    control['disabled'] = true; // control is a protected property only allowed to be set by the cds-control-group
    await componentIsStable(control);
    expect(input.hasAttribute('disabled')).toBe(true);
  });
```

Reply:
OK but as I said in slack the `control['disabled']` is a possibility but do we really want to go into that direction? I had issues with typescript before when doing tests like this. I don't remember what was it but the issues appeared with upgrading to newer version of typescript before.

Reply:
Core spec files have relaxed rules on dynamic property access mainly to set internal states for mocks. However you can avoid this and test this at the group level all the way down to the native input.


```typescript
// control-group.element.spec.ts
it('should set the disabled style on all cds-controls and native inputs if group is disabled', async () => {
  controlGroup.disabled = true;
  await componentIsStable(controlGroup);
  await componentIsStable(controls[0]);
  expect(controls[0].hasAttribute('_disabled')).toBe(true); // cds-control has style hook attr '_disabled'
  expect(controls[0].inputControl.hasAttribute('disabled')).toBe(true); // native html input has attr 'disabled'
});
```